test/e2e/wmco_test.go

```diff
@@ -131,3 +146,68 @@ func testWindowsNodeDeletion(t *testing.T) {
 	}
 	defer testCtx.Cleanup()
 }
+
+// testWMCValidation tests if validations of the fields of WindowsMachineConfigs CRD are working as expected
+func testWMCValidation(t *testing.T) {
+	var wmcReplicasFieldValidationTests = []struct {
+		name                       string
+		isReplicasFieldPresent     bool
+		replicasFieldValue         int
+		isTestExpectedToThrowError bool
+		expectedErrorInTest        string
+	}{
+		// We are only checking negative test cases here, positive test cases would check if custom resource is
+		// getting created as expected and they are handled elsewhere
+		{"replicas field absent", false, 0, false, ""},
```

Comment:
Be explicit about the fields that you want to populate. Please look at example here:

https://github.com/kubernetes/kubernetes/blob/master/pkg/scheduler/framework/plugins/helper/node_affinity_test.go#L34=#L38

Reply:
Done